server.py

```
@@ -4,11 +4,17 @@
 import pymysql
 from datetime import datetime
 import subprocess
+from typing import Dict, List, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)